src/routes/userRoutes.test.js

```diff
@@ -0,0 +1,93 @@
+jest.mock('mailgun-js')
+jest.mock('../../lib/users')
+jest.mock('../../sequelize/db')
+
+const db = require('../../sequelize/db')
+const {resetPassword} = require('./userRoutes')
+const {sendPasswordResetEmail} = require('../../lib/users')
+
+const mockJson = jest.fn()
+
+const res = {
+  status: jest.fn().mockReturnValue({json: mockJson})
+}
+
+describe('Testing user routes', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+  test('should send 400 error if req body does not have email', async () => {
+    const req = {
+      body: {}
+    }
+
+    await resetPassword(req, res)
+    expect(res.status.mock.calls[0][0]).toEqual(400)
+    expect(mockJson.mock.calls[0][0]).toEqual({error: {message: "invalid input"}})
+  })
+
+  test('Should send 400 error if user account does not exist', async () => {
+    db.getModels = jest.fn().mockReturnValue({
+      Accounts: {
+        findOne: jest.fn() 
+      } 
+    }) 
+
+   const req = {
+     body: {
+       email: 'hello@world.com',
+     } 
+   } 
+
+    await resetPassword(req, res)
+    expect(res.status.mock.calls[0][0]).toEqual(400)
+    expect(mockJson.mock.calls[0][0]).toEqual({error: {message: "account does not exist"}})
```

Comment:
![image](https://user-images.githubusercontent.com/61715204/84118569-20b3cc00-aa6e-11ea-8bd7-003aa1e63fb1.png)
This part failed cuz there's a typo in `account`